src/lib/objects/ui.svelte

```diff
@@ -13,56 +18,60 @@
 
 	const component = lookup(objectId)?.standalone
 
+	const wallet = $walletStore.wallet
+	if (!wallet) {
+		// TODO: handle when there is not wallet (redirect to login)
+		throw 'no wallet'
+	}
+	const address = wallet.address
+
+	let store: unknown
+	$: store = $objectStore.objects.get(objectKey(objectId, instanceId))
+	let tokens: Token[]
 	$: tokens = $balanceStore.balances
-	$: nativeToken = tokens.find((token) => token.symbol === 'ETH')
-	$: chat = $chats.chats.get($page.params.id)
-	$: fromUser = chat?.users.find((user) => user.address === $walletStore.wallet?.address)
-	$: toUser = chat?.users.find((user) => user.address !== $walletStore.wallet?.address)
+	let chat: Chat
+	$: chat = $chats.chats.get(chatId) || throwError('chat not found')
```

Comment:
This likely means that on page refresh, it throws error and fails to render the page.